### DISCUSSION TITLE
1.0.17

### DISCUSSION THREAD

**Discussion by codemation** [codemation/easyauth#48] [Announcements]
## What's Changed
* Minify & Refactor Code 🚀  by @yezz123 in https://github.com/codemation/easyauth/pull/37

## New Contributors
* @yezz123 made their first contribution in https://github.com/codemation/easyauth/pull/37

**Full Changelog**: https://github.com/codemation/easyauth/compare/1.0.16...1.0.17

<hr /><em>This discussion was created from the release <a href='https://github.com/codemation/easyauth/releases/tag/1.0.17'>1.0.17</a>.</em>

**Comment by codemation** [comment]
Nice first contribution @yezz123 , looking forward to more in the future!

**Reply by yezz123** [reply]
Welcome, buddy! It was nice to help the project Growup 🚀
